api/controllers/MerchStoreController.ts

```diff
@@ -233,6 +233,7 @@ export class MerchStoreController {
     if (!PermissionsService.canManagePickupEvents(user)) throw new ForbiddenError();
     const pickupEvent = await this.merchStoreService.editPickupEvent(params.uuid,
       editOrderPickupEventRequest.pickupEvent);
-    return { error: null, pickupEvent: pickupEvent.getPublicOrderPickupEvent() };
+    return { error: null,
+      pickupEvent: pickupEvent.getPublicOrderPickupEvent() };
```

Comment:
Noticed that `merchStoreService` does not return a full fetch of `OrderModel[]` for the `order` field, causing `TypeError` to occur whenever we try to render the orders associated with a pickup event (ie `getPublicOrderPickupEvent(true)`. Because of this, an admin cannot directly return the orders present in a pickup event upon edit without needing another query.

Do we need to modify `getPublicOrderPickupEvent` to fix this? Am I misusing the method?

Reply:
> Noticed that merchStoreService does not return a full fetch of OrderModel[] for the order field
I'm not 100% clear on what you mean by this - it looks like `editPickupEvent` calls `orderPickupRepository.findByUuid()`, which does a bunch of joins to get the orders, the orders' items, and the orders' users. If you use `getPublicOrderPickupEvent(true)`, do you still get a `TypeError`?

Reply:
Never mind on this; I had some scratch code that was ruining things, apparently.